### PR TITLE
Add 4.3 latest tags

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #1418

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Comment by vavuthu** [comment]
@petr-balogh : Do we need to change in https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L213

**Comment by petr-balogh** [comment]
> https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L213

We shouldn't IIRC.  We are using this default: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L235 if the `ocs_registry_image` is not specified we are looking for latest tag and using it.

If the `ocs_registry_image` is specified it has basically precedence and then we are not looking for any default value and consuming specified. So that's why I still saw the latest-4.3 instead of particular replaced build tag like `4.3-300.8ec8be3.release_4.3`

**Comment by petr-balogh** [comment]
I see now it's set correctly:
```
13:36:30 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc create -f /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/ocs_ci/templates/ocs-deployment/deploy-with-olm.yaml
13:36:31 - MainThread - ocs_ci.deployment.deployment - INFO - Adding CatalogSource
13:36:31 - MainThread - ocs_ci.utility.templating - INFO - apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  labels:
    ocs-operator-internal: 'true'
  name: ocs-catalogsource
  namespace: openshift-marketplace
spec:
  displayName: Openshift Container Storage
  icon:
    base64data: ''
    mediatype: ''
  image: quay.io/rhceph-dev/ocs-registry:4.3-301.8ec8be3.release_4.3
  publisher: Red Hat
  sourceType: grpc
```

Marking as verified as deployment will fail most likely on just reported deployment blocker BZ: 
https://bugzilla.redhat.com/show_bug.cgi?id=1797590

File: conf/ocsci/ocs-4.3.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 DEPLOYMENT:
-  ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.3"
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.3"
```
Comment (b-ranto) [inline conf/ocsci/ocs-4.3.yaml]: Can you also change this to point to the `ocs-olm-operator` instead of the deprecated `ocs-registry`?

Comment (petr-balogh) [reply conf/ocsci/ocs-4.3.yaml]: Sure, I haven't noticed this and just changed the name of the parameter for this PR. I changed also this in next commit.

**Pull request merged**